### PR TITLE
docs(svg-viewer): migrate story to mdx

### PR DESCRIPTION
**Pull request by polomani**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-541.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-541.gearboxjs.preview.cogniteapp.com

File: src/components/SVGViewer/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+<Props of={SVGViewer} />
+
+## Types
+
+### Conditions
+
+This type describes the parameters the `metadataClassesConditions` function is called with.
+The type can be imported from `@cognite/gearbox`:
+
+```typescript
```
Comment (mrtraser) [inline src/components/SVGViewer/stories/basic.story.mdx]: `typescript` is also not supported yet, use `js` instead

Comment (polomani) [reply src/components/SVGViewer/stories/basic.story.mdx]: fixed

File: src/components/SVGViewer/stories/basic.story.mdx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { SVGViewer } from '@cognite/gearbox';
+```
+
+<Preview>
+  <Story name='Basic Usage'>
+<div style={{height: '100vh'}}>
```
[the remark this answers is not in the document]
Comment (polomani) [reply src/components/SVGViewer/stories/basic.story.mdx]: that's a workaround for typescript source rendered in a wrong way

File: src/components/SVGViewer/stories/basic.story.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+- Locate and zoom onto equipment in the document
+- Search by equipment with locating and zooming results
+
+**NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
+
+## Basic Usage:
+
+Imports:
+
+```typescript jsx
```
[the remark this answers is not in the document]
Comment (polomani) [reply src/components/SVGViewer/stories/basic.story.mdx]: fixed

File: src/components/SVGViewer/stories/basic.story.mdx [hunk 4]
```diff
@@ -0,0 +1,102 @@
+import { Meta, Story, Preview, Props, Source } from '@storybook/addon-docs/blocks';
```
[the remark this answers is not in the document]
Comment (polomani) [reply src/components/SVGViewer/stories/basic.story.mdx]: fixed

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-541.gearboxjs.preview.cogniteapp.com

File: src/components/SVGViewer/stories/basic.story.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (mrtraser) [reply src/components/SVGViewer/stories/basic.story.mdx]: got it

**Comment by polomani** [comment]
Screenshot 2019-12-16 at 15 06 23
Is it the same for you guys? The svg is not centered for some reason, Idk why

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-541.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-541.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-541.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-541.gearboxjs.preview.cogniteapp.com

**Pull request merged**